### PR TITLE
CN Temp mail extention added

### PR DESCRIPTION
**Pull request by immortalsantee**
## Description

- CN Temp mail extension added.

### CODE REVIEW

File: extensions/cn-temp-mail/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "cn-temp-mail",
+  "title": "Temp Mail - CN",
+  "description": "It creates temporary email.",
```
Comment (pernielsentikaer) [inline extensions/cn-temp-mail/package.json:5]:
```suggestion
  "name": "maildrop",
  "title": "Maildrop - Create Temporary Email",
  "description": "Provides free disposable e-mail addresses for use in web forms, app signups, or any other place you'd like to protect your privacy.",
```
And change folder name

File: extensions/cn-temp-mail/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "icon": "command-icon.png",
+  "author": "immortalsantee",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Create Temp Mail - CN",
```
Comment (pernielsentikaer) [inline extensions/cn-temp-mail/package.json:15]:
```suggestion
      "title": "Create a Temporary E-mail",
      "subtitle": "Maildrop",
```

File: extensions/cn-temp-mail/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      "description": "Generate temporary email.",
```
Comment (pernielsentikaer) [inline extensions/cn-temp-mail/package.json:16]:
```suggestion
      "description": "Generates a temporary email and copy it to clipboard",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪

Works as expected, I have a few comments and suggestions:

- Could also be a `no-view` command with an argument, maybe that's faster? Let me know if you need some more explanation 😊

- Maybe use the logo from maildrop instead?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by koinzhang** [comment]
Hi @immortalsantee @pernielsentikaer Similar extensions already exist in the store. I checked the source code and your extension api and function are the same as the [Temporary Email](https://www.raycast.com/natterstefan/temporary-email) extension is the same
image

**Comment by immortalsantee** [comment]
> Hi @immortalsantee @pernielsentikaer Similar extensions already exist in the store. I checked the source code and your extension api and function are the same as the [Temporary Email](https://www.raycast.com/natterstefan/temporary-email) extension is the same
> image

Yes @pernielsentikaer, but I wanted to have a choice to create my own username while using temp mail. And in some cases, using random UUID as username to generate temporary email, doesn't work as the website shows error that the email is not valid. Thus, I created this extension.

**Comment by pernielsentikaer** [comment]
Thanks for pointing out @koinzhang, I'm not sure why I didn't find it while checking 😊

@immortalsantee it would be better to make a PR that enhances the other one, maybe make arguments so you can offer named inboxes 💯

**Pull request closed without merging**